ee/control/bucket_consumer.go

```diff
@@ -28,6 +28,10 @@ func NewBucketConsumer(logger log.Logger, db *bbolt.DB, bucketName string) *buck
 }
 
 func (bc *bucketConsumer) Update(data io.Reader) error {
+	if bc == nil {
+		return fmt.Errorf("bucketConsumer is nil")
```

Comment:
Should these (here, line 99, line 119) be errors.New instead since there's no string interpolation happening? Same for the addition to NotificationConsumer.Update

Reply:
thanks, fixed!